features/networking/multus.feature

```diff
@@ -160,7 +155,7 @@ Feature: Multus-CNI related scenarios
 
     # Create the second pod which consumes the macvlan cr
     When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/1interface-macvlan-vepa.yaml" replacing paths:
-      | ["spec"]["nodeName"] | "<%= cb.pod_node %>" |
+      | ["spec"]["nodeName"] | "<%= cb.target_node %>" |    
```

Comment:
Useless spaces added at the end of line.

Reply:
Thanks @liangxia